PS8lite.Rmd

```diff
@@ -187,10 +189,39 @@ MiscVal + MoSold + YrSold" %>%
 Fit one Random Forest model `model_rf` using any approach you want.
 
 ```{r}
+# Define cross-validation settings: 10-fold CV
+fit_control <- trainControl(method = "cv", number = 10)
 
+# Setup search grid of "tuning" parameters
+mtry <- 2:5 #NOTE: I wanted to use all 36 predictors (mtry <- 2:36), but it took way too long to compute and also crashed my RStudio. So I'm sticking with 2-5 predictors just so it will run! 
```

Comment:
Indeed! This is a great example of a problem where computational constraints become an issue! In other words, method A might yield "better predictions" than method B, but if method A takes a week to run and method B takes only a few minutes, you might sacrifice a little predictive power for speed!